hrm-form-definitions/form-definitions/uscr/v1/CaseSections.json

```
@@ -7,6 +7,10 @@
     "label": "Incident Report",
     "formPath": "caseForms/IncidentReportForm.json"
   },
+  "assignedResponders": {
```

```suggestion
  "assignedResponder": {
```
We use singular case for section types as a convention